cli/tests/pcluster/config/test_source_consistency.py

```diff
@@ -36,13 +36,6 @@ def test_mapping_consistency():
 
         for param_key, param_definition in section_definition.get("params").items():
 
-            # Boolean params must have a default value
```

Comment:
Boolean parameters no longer are required to have a default value. This was necessary because the new `copy_tags_to_backup` parameter can't have a default value, because a value specified for this parameter is interpreted in one of the new validators as a sign that the user is trying to enable automatic backup functionality.